frontend/pages/beta/index.tsx

```diff
@@ -137,25 +138,24 @@ export default function ExploreModels() {
             <div data-test='modelListBox'>
               {(!models || models.length === 0) && <EmptyBlob data-test='emptyModelListBlob' text='No models here' />}
               {models &&
```

Comment:
`models &&` can be removed if you default to an array as suggested in another comment